olive/evaluator/accuracy.py

```diff
@@ -77,15 +77,15 @@ def prepare_tensors(preds, target, dtypes=torch.int):
         return preds, target
 
     @abstractmethod
-    def measure(self, preds, target):
+    def measure(self, model_output, target):
         raise NotImplementedError()
 
 
 class AccuracyScore(AccuracyBase):
     name: str = "accuracy_score"
 
-    def measure(self, preds, target):
-        preds_tensor, target_tensor = self.prepare_tensors(preds, target)
+    def measure(self, model_output, target):
+        preds_tensor, target_tensor = self.prepare_tensors(model_output[0], target)
```

Comment:
I think it is clearer if we access it using `model_outputs.preds`

Reply:
The main concern is about compatibility on tuple and named tuple. Once user want use Olive's evaluator, there is no need to pack the (preds, logits) with pre-defined name.

Reply:
But it may only appear on the case user rewrite the inference function which is not supported in Olive now. 

Updated with named tuple.